<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Shp2PgsqlGUI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="2" column="0" colspan="4">
     <widget class="QTableWidget" name="tbl_file">
      <column>
       <property name="text">
        <string>Source</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Schema</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Table</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Geom Column</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>SRID</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="lbl_log">
      <property name="text">
       <string>Log</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="4">
     <widget class="QPlainTextEdit" name="txt_log">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="btn_add">
      <property name="text">
       <string>Add File</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="lbl_file_list">
      <property name="text">
       <string>File list</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="btn_clear">
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2" colspan="2">
     <widget class="QWidget" name="widget_tr" native="true">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="lbl_user">
         <property name="text">
          <string>Username</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="fld_user">
         <property name="placeholderText">
          <string>username</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lbl_password">
         <property name="text">
          <string>Password</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="fld_password">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
         <property name="placeholderText">
          <string>password</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QPushButton" name="btn_test">
         <property name="text">
          <string>Test Connection</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QWidget" name="widget_tl" native="true">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="lbl_host">
         <property name="text">
          <string>Host</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="fld_host">
         <property name="text">
          <string>localhost</string>
         </property>
         <property name="placeholderText">
          <string>host</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lbl_port">
         <property name="text">
          <string>Port</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="fld_port">
         <property name="text">
          <string>5432</string>
         </property>
         <property name="placeholderText">
          <string>port</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lbl_dbname">
         <property name="text">
          <string>Database</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="fld_dbname">
         <property name="placeholderText">
          <string>database</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="6" column="2" colspan="2">
     <widget class="QPushButton" name="btn_import">
      <property name="text">
       <string>Import</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QPushButton" name="btn_save_log">
      <property name="text">
       <string>Save Log</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="act_add"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="act_about"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Help"/>
  </widget>
  <action name="act_add">
   <property name="text">
    <string>&amp;Add File</string>
   </property>
  </action>
  <action name="act_about">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>fld_host</tabstop>
  <tabstop>fld_port</tabstop>
  <tabstop>fld_dbname</tabstop>
  <tabstop>fld_user</tabstop>
  <tabstop>fld_password</tabstop>
  <tabstop>btn_test</tabstop>
  <tabstop>tbl_file</tabstop>
  <tabstop>btn_add</tabstop>
  <tabstop>btn_clear</tabstop>
  <tabstop>txt_log</tabstop>
  <tabstop>btn_save_log</tabstop>
  <tabstop>btn_import</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
